### DISCUSSION TITLE
`yarn create next-app` fail with node 16.x.x

### DISCUSSION THREAD

**Discussion by toviszsolt** [vercel/next.js#31307] [Help] [answered]
### What version of Next.js are you using?

not necessary

### What version of Node.js are you using?

16.x.x

### What browser are you using?

not necessary

### What operating system are you using?

not necessary

### How are you deploying your application?

not necessary

### Describe the Bug

`yarn create next-app` fail with node 16.x.x

Output:
```
> yarn create next-app
yarn create v1.22.15
[1/4] Resolving packages...
[2/4] Fetching packages...
error create-strapi-app@3.4.3: The engine "node" is incompatible with this module. Expected version ">=10.16.0 <=14.x.x". Got "16.13.0"
error Found incompatible module.
info Visit https://yarnpkg.com/en/docs/cli/create for documentation about this command.
```

### Expected Behavior

Running successfully? :P --  Sorry for this (required field)

### To Reproduce

`yarn create next-app`

**Comment by stefanprobst** [comment]
> error create-strapi-app@3.4.3: The engine "node" is incompatible with this module

looks like that's an issue with `create-strapi-app` not `create-next-app`?

**Comment by toviszsolt** [comment]
There is new version of `create-strapi-app`. Current version is `3.6.8`. Maybe a simple package upgrade will solve the issue. Now I just checked, upgrade won't solve the issue, they don't keep up to date the `package.json.`

```
"version": "3.6.8"
...
"engines": {
    "node": ">=10.16.0 <=14.x.x",
    "npm": ">=6.0.0"
  },
```

**Comment by timneutkens** [comment, marked as answer]
Next.js does not depend on `create-strapi-app` and is fully compatible with Node.js 16 (and 17), what is happening here is that yarn is checking all globally installed deps during installation and that fails. You'll want to remove `create-strapi-app` using `yarn global remove create-strapi-app` I guess. Going to convert this to discussions as it's not a bug in Next.js.

**Reply by toviszsolt** [reply]
Confirmed. This is anomaly of yarn.

**Reply by mateus-sartorio** [reply]
Thanks bro, worked for me

**Comment by ragnarok22** [comment]
Also fails on Node 14.15.4. I think is because of eslintrc

`error @eslint/eslintrc@1.0.5: The engine "node" is incompatible with this module. Expected version "^12.22.0 || ^14.17.0 || >=16.0.0". Got "14.15.4"`

**Reply by toviszsolt** [reply]
@ragnarok22 yes, it's `eslint` engine declaration problem, so you can write issue here: [eslint/eslintrc](https://github.com/eslint/eslintrc).

**Comment by destefanis** [comment]
How should developers create a new next.js app then in the meantime? Since this command won't work should we use the manual method?

**Reply by destefanis** [reply]
Just needed up to update to the latest version of node to avoid the eslint issue.
